docs/copied-from-beats/docs/security/users.asciidoc

```diff
@@ -197,156 +344,245 @@ information.
 |Send monitoring data to the monitoring cluster
 |====
 
-. Assign the following role to users who will view the monitoring data in
-{kib}:
+TIP: See <<monitoring-metricbeat-collection>>
+for complete details on setting up Metricbeat collection.
+--
+
+[float]
+[[privileges-to-publish-monitoring-view]]
+==== View monitoring data
+
+To grant users the required privileges for viewing monitoring data:
+
+. Create a *monitoring role*, called something like
++{beat_default_index_prefix}_monitoring_viewer+, that has the following privileges:
++
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| Spaces
+|`Read` on Stack monitoring
+|Read-only access to the Stack Monitoring feature in {kib}.
+
+| Spaces
+|`Read` on Dashboards
+|Read-only access to the Dashboards feature in {kib}.
+|====
++
+. Assign the *monitoring role*, along with the following built-in roles, to users who
+need to view monitoring data for {beatname_uc}:
 +
 [options="header"]
 |====
 |Role | Purpose
 
 |`monitoring_user`
-|Use *Stack Monitoring* in {kib} to monitor {beatname_uc}
+|Grants access to monitoring indices for {beatname_uc}
 |====
---
-endif::serverless[]
 
-[[privileges-to-publish-events]]
-==== Grant privileges and roles needed for publishing
+////
+***********************************  ***********************************
+***********************************  ***********************************
+////
 
-Users who publish events to {es} need to create and write to {beatname_uc}
-indices. To minimize the privileges required by the writer role, use the
-<<privileges-to-setup-beats,setup role>> to pre-load dependencies. This section
-assumes that you've pre-loaded dependencies. 
+[[kibana-user-privileges]]
+=== Grant privileges and roles needed to read APM data in {kib}
```

Comment:
Same comment as above - it would be nice to move this to Kibana in the long run.

Reply:
I agree! Just trying to see everything in one place to ensure consistency before breaking it up.

Reply:
And actually, let's do this now.